seed/tests/test_admin_views.py

```diff
@@ -110,7 +110,7 @@ def test_add_user_existing_org(self):
             'first_name': 'New',
             'last_name': 'User',
             'email': 'new_user@testserver',
-            'role_level': 'ROLE_MEMBER',
+            'role': 'member',
```

Comment:
This test has been broken for a while, but due to a few stacked errors it 'passed'. In /v3/users.py, the `def create` function first calls `org.add_member` without passing a `role` argument triggering the default argument 'owner'. The role passed in the request is applied after the user is created `if body.get('role')...`. Because 'role' was not a key this code was skipped and the new user remained an 'owner'